charts/kafka-connect/values.yaml

```diff
@@ -0,0 +1,280 @@
+prometheus:
+  ## JMX Exporter Configuration
+  ## ref: https://github.com/prometheus/jmx_exporter
+  jmx:
+    ## @param prometheus.jmx.enabled JMX Exporter
+    enabled: true
+
+    ## @param prometheus.jmx.image.registry JMX Exporter image registry
+    ## @param prometheus.jmx.image.repository JMX Exporter image repository
+    ## @param prometheus.jmx.image.tag JMX Exporter image tag
+    ## @param prometheus.jmx.image.pullPolicy JMX Exporter image pull policy
+    ## @param prometheus.jmx.image.pullSecrets JMX Exporter image pull secrets
+    image:
+      registry: docker.io
+      repository: solsson/kafka-prometheus-jmx-exporter@sha256
+      tag: 6f82e2b0464f50da8104acd7363fb9b995001ddff77d248379f8788e78946143
+      ## Specify a imagePullPolicy
+      ## ref: https://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+      ##
+      pullPolicy: IfNotPresent
+      ## Optionally specify an array of imagePullSecrets.
+      ## Secrets must be manually created in the namespace.
+      ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+      ## e.g:
+      ## pullSecrets:
+      ##   - myRegistryKeySecretName
+      ##
+      pullSecrets: []
+    ## @param prometheus.jmx.resources JMX Exporter resources
+    ## Resources configuration for the JMX exporter container.
+    ## See the `resources` documentation above for details.
+    resources: {}
+
+## You can list load balanced service endpoint, or list of all brokers (which is hard in K8s).  e.g.:
+## @param kafka.bootstrapServers list of the kafka brokers ex."PLAINTEXT://dozing-prawn-kafka-headless:9092"
+kafka:
+  bootstrapServers: ""
+
+## @param schema-registry.url schema-registry url
+# If the Kafka Chart is disabled a URL and port are required to connect
+## e.g. gnoble-panther-schema-registry:8081
+schema-registry:
+  url: ""
+
+## @param volumeMounts List of volumeMounts for connect server container
+## ref: https://kubernetes.io/docs/concepts/storage/volumes/
+volumeMounts:
+# - name: credentials
+#   mountPath: /etc/creds-volume
+
+## @param volumes List of volumeMounts for connect server container
+## ref: https://kubernetes.io/docs/concepts/storage/volumes/
+volumes:
+# - name: credentials
+#   secret:
+#     secretName: creds
+
+## @param secrets Secret with multiple keys to serve the purpose of multiple secrets
+## Values for all the keys will be base64 encoded when the Secret is created or updated
+## ref: https://kubernetes.io/docs/concepts/configuration/secret/
+secrets:
+  # username: kafka123
+  # password: connect321
+
+## @param livenessProbe the livenessProbe for the kafka connect pods
+## "livenessProbe" is required only for the edge cases where the custom script to be ran takes too much time
+## and errors by the ENTRYPOINT are ignored by the container
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+livenessProbe:
```

Comment:
liveness/readiness probes should be enabled by default

Reply:
done